packages/tasit-act/src/ethers.test.js

```diff
@@ -0,0 +1,92 @@
+import { expect, assert } from "chai";
+import { ethers } from "ethers";
+// Note: This file is originally genarated by `tasit-contracts` and was pasted here manually
+// See https://github.com/tasitlabs/TasitSDK/issues/45
+import { abi as contractABI } from "./helpers/SimpleStorage.json";
```

Comment:
Should we import a human-readable ABI instead? Maybe it's useful to have both versions - tests for a full ABI and a human-readable one.

Reply:
Hmmm. I've thought that will never use human-readable ABI on real test cases, I've deleted it off because of that.

Reply:
I'll close this thread and we can discuss here https://github.com/tasitlabs/TasitSDK/pull/71#discussion_r243323261